ostd/src/cpu/cpu_local.rs

```diff
@@ -115,6 +122,11 @@ impl<T> CpuLocal<T> {
     /// This function calculates the virtual address of the CPU-local object based on the per-
     /// cpu base address and the offset in the BSP.
     fn get(&self) -> *const T {
+        // CPU-local objects should be initialized before being accessed. It should be ensured
+        // by the implementation of OSTD initialization.
+        #[cfg(debug_assertions)]
+        debug_assert!(IS_INITIALIZED.load(Ordering::Acquire));
```

Comment:
`IS_INITIALIZED` is not used for synchronization, so I think that all accesses to it should use `Ordering::Relaxed`.